framework/aster-frame/src/lib.rs

```diff
@@ -15,6 +15,7 @@
 #![feature(panic_info_message)]
 #![feature(ptr_sub_ptr)]
 #![feature(strict_provenance)]
+#![feature(specialization)]
```

Comment:
Note that the feature [`specialization`](https://github.com/rust-lang/rust/issues/31844) remains an incomplete feature for 8 years. The warning is suppressed by `#![allow(incomplete_features)]`. Consider using [`min_specialization`](https://github.com/rust-lang/rust/pull/68970) on new code. If not possible, add more comments below.